contrib/babelfishpg_tsql/src/hooks.c

```diff
@@ -1330,105 +1379,40 @@ pre_transform_target_entry(ResTarget *res, ParseState *pstate,
 		 */
 		if (alias_len > 0)
 		{
-			char	   *alias = palloc0(alias_len + 1);
-			bool		dq = *colname_start == '"';
-			bool		sqb = *colname_start == '[';
-			bool		sq = *colname_start == '\'';
-			bool		identifier_truncated = false;
-			const char *colname_end;
-			bool		enc_is_single_byte;
-			enc_is_single_byte = pg_database_encoding_max_length() == 1;
+			char	*alias = palloc0(alias_len + 1);
+			const char	*original_name = NULL;
+			int		actual_alias_len = 0;
+			
+			/* To handle queries like SELECT ((<column_name>)) from <table_name> */
+			while(*colname_start == '(' || *colname_start == ' ')
```

Comment:
please add some more test cases for this change.

Reply:
Resolved.